doc/man7/scrypt.pod

```diff
@@ -136,19 +88,20 @@ This example derives a 64-byte long test vector using scrypt using the password
 
  EVP_PKEY_CTX_free(pctx);
 
-=head1 CONFORMING TO
-
-RFC 7914
```

Comment:
Why remove this?

Reply:
I move it to `doc/man3/EVP_PKEY_CTX_set_scrypt_N.pod`...  frankly, I'm unsure if this kind of section belongs in section 3 or 7...  what are your thoughts?

Reply:
It seems to make more sense to me in 7. Its the implementation as a whole that conforms, not individual functions. But OTOH I don't see great harm in having it in both.

Reply:
I see possible confusion with having it in both as soon as someone updates only one of them.  I prefer not to encourage that.

Moving them to the section 7 files.